### PR TITLE
Feat/add custom dapr components

### PR DESCRIPTION
**Pull request by m-voit**
## Describe your Pull Request
- Add custom dapr components to use different state store and pubsub components depending on the environment (e.g cloud or local).
- Add pwsh scripts to start daprized services with cloud or local components.
- Align naming of dapr components and daprized apps.

Fixes # (issue)

Please select relevant options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: services/client/main.go [hunk 1]
```diff
@@ -19,20 +19,20 @@ func main() {
 
 	nico := &dapr.DataContent{
 		ContentType: "application/json",
-		Data:        []byte(`{ "Id": 1, "Name": "Fred Duck" }`),
+		Data:        []byte(`{ "Id": 1, "Name": "Nico" }`),
```
Comment (mjisaak) [inline services/client/main.go]: Ich würde den Namen Fred Duck und John Doe bevorzugen allerdings sollten wir die variablen (nico und martin) ändern

File: services/messageservice/Services/MessageService.cs [hunk 2]
```diff
@@ -17,7 +17,7 @@ namespace Demo.Dapr.MessageService
     public class MessageService : AppCallback.AppCallbackBase
     {
         readonly JsonSerializerOptions jsonOptions = new JsonSerializerOptions { PropertyNamingPolicy = JsonNamingPolicy.CamelCase };
-        public const string StoreName = "statestore";
+        public const string StoreName = "dapr-demo-state-store";
```
Comment (mjisaak) [inline services/messageservice/Services/MessageService.cs:20]: Sehr schön, dass Du die Komponenten umbenannt hast

**Review by mjisaak — CHANGES_REQUESTED** [review]
Sieht gut aus!

File: services/client/main.go (hunk 1 above)
Comment (m-voit) [reply services/client/main.go]: Alles klar, ich habe die Namen und die Variablen entsprechend angepasst.

**Review by mjisaak — APPROVED** [review]

**Pull request merged**